cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/http/HttpBackend.java

```diff
@@ -149,7 +150,7 @@ public JsonResponse doRequest(String method, String url, boolean relative, Array
         } // if else
     } // doRequest
         
-    private JsonResponse doRequest(String method, String url, ArrayList<Header> headers, StringEntity entity)
+    protected JsonResponse doRequest(String method, String url, ArrayList<Header> headers, StringEntity entity)
```

Comment:
Javadoc is required. Something saying what the methods does, and why it is protected (it is protected becuase it is used by the tests).